akka-stream-tests/src/test/scala/akka/stream/scaladsl/FutureFlattenSpec.scala

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (C) 2015-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.scaladsl
+
+import scala.concurrent.{ Await, ExecutionContext, Future, Promise }
+import scala.concurrent.duration._
+
+import java.util.concurrent.CompletionStage
+import scala.compat.java8.FutureConverters._
+
+import akka.NotUsed
+import akka.stream._
+import akka.stream.stage.{ GraphStage, GraphStageLogic, OutHandler }
+import akka.stream.scaladsl._
+import akka.stream.impl.fusing.GraphStages.FutureFlattenSource
+
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+
+import akka.testkit.EventFilter
+import akka.stream.testkit.{ StreamSpec, TestSubscriber }
+import akka.stream.testkit.Utils.assertAllStagesStopped
+
+class FutureFlattenSpec extends StreamSpec {
+  val materializer = ActorMaterializer()
```

Comment:
Make this one implicit as it is anyway what is used in all tests, and remove the extra curly brackets on line 28, and the individual implicits further down. You can do the same with the execution context